.gitignore

```diff
@@ -36,4 +36,4 @@ solutions/**/*.tar.gz
 solutions/tModLoader v*
 solutions/Release/
 **/msbuild.binlog
-/steam_build
+/steam_build
```

Comment:
accidental removal of newline?

Reply:
Well, I even checked out the .gitignore file from 1.4.4 and pushed it, to get rid of this. It still shows up though.

Reply:
Nevermind it was just Github unwilling to update the page correctly...